test/users/engin/refComparison.skipif

```diff
@@ -0,0 +1 @@
+CHPL_COMM!=none
```

Comment:
Hmm.. not that it fails with `CHPL_COMM!=none`. Just that the on statement is optimized away (I think) with that environment, thus it doesn't test anything new. In other words, with `CHPL_COMM=none` this test passes without this PR, too.

I can remove it if you want.